local-test.js

```diff
@@ -0,0 +1,43 @@
+const fs = require('fs')
+const Firestore = require('./utils/firestore.js')
+const config = require('config')
+
+// check whether github credentials are not falsy
+if (
+  !config.githubOauth ||
+  !config.githubOauth.clientId ||
+  !config.githubOauth.clientSecret
+) {
+  throw new Error('Github credentials are not properly set')
+}
+
+// throw an error if unable to read file
+try {
+  var firestoreData = fs.readFileSync('./firestore-private-key.json', 'utf8')
+} catch (error) {
+  throw new Error('Please make sure firestore-private-key.json file is correct')
+}
+
+// check whether firestoreData is empty, null, and  undefined
+if (!firestoreData) {
+  throw new Error('Please make sure firestore-private-key.json file is not empty')
+}
+
+readWriteCheck()
+
+// check local development have permission to read and write in firestore or not
+async function readWriteCheck () {
```

Comment:
Can we use arrow functions here? Is eslint setup for the repository?

Reply:
Same.